deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -23,19 +23,19 @@ spec:
       priorityClassName: csi-gce-pd-controller
       containers:
         - name: csi-provisioner
-          image: gke.gcr.io/csi-provisioner
+          image: gcr.io/k8s-sig-storage/csi-provisioner
```

Comment:
you'll need to use the "k8s.gcr.io/sig-storage" alias to so that the image transformation matches

Reply:
I changed them all to the alias, and forgot to save the changes to this file. D'oh